assets/js/admin/orders/list-table.js

```diff
@@ -7,10 +7,11 @@ import { jQueryReady } from 'utils/jquery.js';
 
 jQueryReady( () => {
 
-	$( '.edd-advanced-filters-button' ).on( 'click', function( e ) {
-		e.preventDefault();
-
-		$( '#edd-advanced-filters' ).toggleClass( 'open' );
-	} );
+	$( '.download_page_edd-payment-history .row-actions .delete a' ).on( 'click', function() {
+		if( confirm( edd_vars.delete_order ) ) {
```

Comment:
If the key is being reverted to `delete_payment` does this need to be changed also?

Reply:
🤦‍♀️ Yep.